common/script/ops/openMysteryItem.js

```diff
@@ -28,17 +27,13 @@ module.exports = function openMysteryItem (user, req = {}, analytics) {
     });
   }
 
-  if (typeof window !== 'undefined') {
-    if (!user._tmp) user._tmp = {};
-    user._tmp.drop = item;
-  }
-
   if (req.v2 === true) {
     return user.items.gear.owned;
   } else {
     return [
       user.items.gear.owned,
       i18n.t('mysteryItemOpened', req.language),
+      item,
```

Comment:
I see that not returning the item in first place wasn't very smart on my side... but now changing the response to [{item, owned}, message] would break compatibility